packages/ui/src/utils.ts

```diff
@@ -0,0 +1,130 @@
+export const isNil = <T>(
```

Comment:
can be extracted to a utils package

Reply:
And we can remove many of this ones if not used